nilearn/glm/first_level/hemodynamic_models.py

```diff
@@ -724,7 +724,7 @@ def compute_regressor(
     oversampling : int, default=50
         Oversampling factor to perform the convolution.
 
-    fir_delays : [int] 1D-array-like, optional
+    fir_delays : [int] 1D-array-like, default=None
```

Comment:
```suggestion
    fir_delays : [int] 1D-array-like or None, default=None
```

In this case could also update the doc string to also mention.

```rst
Will be set to ``[0]`` if ``None`` is passed.
```